sources/niUI/src/GDRV_GLES2.cpp

```diff
@@ -4575,25 +4596,31 @@ struct cGLES2GraphicsDriver : public cIUnknownImpl<iGraphicsDriver>
         mCache.ShouldUpdate(&bShouldUpdate,eGLCache_RasterizerDepthOnly,
                             niFlagIs(matFlags,eMaterialFlags_DepthOnly));
         mCache.ShouldUpdate(&bShouldUpdate,eGLCache_RasterizerFlippedRT,isFlippedRT);
+        mCache.ShouldUpdate(&bShouldUpdate, eGLCache_ReversedCulling,
+                            isReversedCulling);
         if (bShouldUpdate)
         {
           iRasterizerStates* pRSStates = mpGraphics->GetCompiledRasterizerStates(hRS);
+
+          sRasterizerStatesDesc v =
+              *(sRasterizerStatesDesc *)pRSStates->GetDescStructPtr();
+
+          eCullingMode cullingMode = isReversedCulling
+                                         ? _ReverseCulling(v.mCullingMode)
+                                         : v.mCullingMode;
```

Comment:
Shouldnt this just be in GL_ApplyRasterizerStates ?

Reply:
TOTALLY that was kind of a brain fart.